### ISSUE TITLE
Error 500 field of type bit

### ISSUE THREAD

**Issue by gmrfrost** [phpmyadmin/phpmyadmin#16048]
Hi,

We are facing an error when trying to modify a field of type BIT. 
phpMyAdmin-5.0.2-1
php73-php-common-7.3.16-1 (FPM)
Linux  4.4.217-1 #1 SMP  2020 x86_64 x86_64 x86_64 GNU/Linux
CentOS release 6.10 (Final)
Server version: Apache/2.4.34 (Red Hat) (Server MPM:     event)

Trace:
 ```
PHP Fatal error:  Uncaught TypeError: htmlspecialchars_decode() expects parameter 1 to be string, null given in /usr/share/phpMyAdmin/libraries/classes/Util.php:2863
Stack trace:
#0 /usr/share/phpMyAdmin/libraries/classes/Util.php(2863): htmlspecialchars_decode(NULL, 3)
#1 /usr/share/phpMyAdmin/libraries/tbl_columns_definition_form.inc.php(339): PhpMyAdmin\Util::convertBitDefaultValue(NULL)
#2 /usr/share/phpMyAdmin/libraries/classes/Controllers/Table/StructureController.php(573): include('/usr/share/phpM...')
#3 /usr/share/phpMyAdmin/libraries/classes/Controllers/Table/StructureController.php(218): PhpMyAdmin\Controllers\Table\StructureController->displayHtmlForColumnChange(Array, 'tbl_structure.p...', Object(Symfony\Component\DependencyInjection\ContainerBuilder))
#4 /usr/share/phpMyAdmin/tbl_structure.php(68): PhpMyAdmin\Controllers\Table\StructureController->indexAction(Object(Symfony\Component\DependencyInjection\ContainerBuilder))
#5 {main}
  thrown in /usr/share/phpMyAdmin/libraries/classes/Util.php on line 2863
```

Regards,

**Comment by ibennetch** [comment]
Thanks for the report.

This affects QA_5_0, does not affect QA_4_9

**Comment by peterkroon** [comment]
I can reproduce this only when the default value is set to NULL. When a default value has been specified then I'm able to edit the bit field.

Server version: 10.1.44-MariaDB-0+deb9u1 - Debian 9.11
nginx/1.10.3
Database client version: libmysql - mysqlnd 7.4.4
PHP version: 7.4.4
Version information: 5.0.2 (up to date)

**Comment by williamdes** [comment]
ae640ebcce4fffdc489f3abbdaf53353e4c75e8e is the fix and I added a test case aab8c19731b54d629a669ab97c3be01fb495915f
